hack/lib/serverless.bash

```diff
@@ -347,7 +347,7 @@ function teardown_serverless {
     oc delete knativekafka.operator.serverless.openshift.io knative-kafka -n "${EVENTING_NAMESPACE}"
   fi
   logger.info 'Ensure no knative eventing or knative kafka pods running'
-  timeout 600 "[[ \$(oc get pods -n ${EVENTING_NAMESPACE} --field-selector=status.phase!=Succeeded -o jsonpath='{.items}') != '[]' ]]"
+  timeout 700 "[[ \$(oc get pods -n ${EVENTING_NAMESPACE} --field-selector=status.phase!=Succeeded -o jsonpath='{.items}') != '[]' ]]"
```

Comment:
let's still break into different jobs, ok?

Reply:
That's for a different task/subject https://github.com/openshift-knative/serverless-operator/pull/1674 I guess, right?

Reply:
but yes